frontend/tsconfig.json

```diff
@@ -35,4 +36,4 @@
         "watchFile": "useFsEvents",
         "fallbackPolling": "dynamicpriority"
     }
-}
\ No newline at end of file
+}
```

Comment:
[nit]: could you make your editor add EOF line, so there are no formatting changes.

Reply:
I have `"files.insertFinalNewline": true` because there should be an EOF line always, now it's there (previously there wasn't EOF character)